command/monitor.go

```diff
@@ -40,21 +40,26 @@ var MonitorEnv = cli.Command{
 			color.Red("You should specify an existing env name")
 			return nil
 		}
+		if len(c.Args()) == 2 {
+			color.Red("You should specify a build id")
+			return nil
```

Comment:
Same as you're doing on other places that return should be a `os.Exit(1)`. Maybe we could just create a method like
```
func PrintError(msg string) {
    color.Red(msg)
    os.Exit(1)
}
```
So all errors are managed from the same place, i guess we can create a separate issue to implement it, what do you think?

Reply:
Issue created: https://github.com/ernestio/ernest/issues/701